mne/tests/test_evoked.py

```diff
@@ -759,3 +761,24 @@ def test_hilbert():
 
 
 run_tests_if_main()
+
+
+def fun(evoked_data):
+    """Auxiliary function to test apply_function."""
+    mult_array = np.random.rand(1000)
+    product = np.multiply(mult_array, evoked_data)
+    return np.asarray(product)
+
+
+def test_apply_function_evk():
+    """Check the apply_function method for evoked data."""
+    # create fake evoked data to use for checking apply_function
+    data = np.random.rand(2, 10, 1000)
+    events = np.array([[0, 0, 1], [INT32_MAX, 0, 2]])
+    info = mne.create_info(10, 1000., 'eeg')
+    epochs = mne.EpochsArray(data, info, events)
+    evoked = epochs.average(method='mean')
```

Comment:
why not create the evoked directly using `EvokedArray`?  Also why do you need events here?